Makefile

```diff
@@ -120,6 +122,16 @@ gen_patch_iscp:
 gen_patch_values:
 	diff -u pkg/apis/istio/v1alpha2/values/values_types.pb.go.orig pkg/apis/istio/v1alpha2/values/values_types.pb.go > pkg/apis/istio/v1alpha2/values/fix_values_structs.patch || true
 
+sync_charts:
+	cp -Rf $(charts_dir)/crds $(out_dir)
```

Comment:
need to add step to clone installer repo firstly?

Reply:
Done, I also just moved it to a script since it should always be called with a branch name.